core/src/test/java/org/elasticsearch/monitor/os/OsProbeTests.java

```diff
@@ -47,12 +41,13 @@ public void testOsStats() {
         OsStats stats = probe.osStats();
         assertNotNull(stats);
         assertThat(stats.getTimestamp(), greaterThan(0L));
+        assertThat(stats.getCpu().getPercent(), anyOf(equalTo((short) -1), is(both(greaterThanOrEqualTo((short) 0)).and(lessThanOrEqualTo((short) 100)))));
```

Comment:
At this time, I think that it's available on all the systems (it appears to be available on Linux, OS X, and Windows). I don't think such a catch is necessary, at least for daily development and the CI infrastructure.